packages/cli/src/models/files/ZosNetworkFile.ts

```diff
@@ -72,6 +72,11 @@ export default class ZosNetworkFile {
     dependencies: { [dependencyName: string]: DependencyInterface };
   };
 
+  public static getZosversion(fileName: string): string | null {
```

Comment:
can we use simply `network` here instead

Reply:
okabz